generator/generate-one.sh

```diff
@@ -30,32 +33,13 @@ echo "Client Library ArtifactId: $client_lib_artifactid";
 WORKING_DIR=`pwd`
 
 
-# get googleapis repo
-git clone https://github.com/googleapis/googleapis.git
-
-# Prepare `gapic-generator-java` with Spring generation ability.
-# If keeping a copy in this repo, this is not needed.
-# Checkout `gapic-generator-java`
-git clone https://github.com/googleapis/gapic-generator-java.git
-# get into gapic and checkout branch to use
-cd gapic-generator-java
-git checkout autoconfig-gen-draft2
-# go back to previous folder
-cd -
-
-
-cd googleapis
-# fix googleapis committish for test/dev purpose
-git checkout f88ca86
-# todo: change to local repo --> gapic
-# very not stable change - todo: change to search and replace.
-sed -i '274,278d' WORKSPACE
-
-sed -i '274 i local_repository(\n    name = "gapic_generator_java",\n    path = "../gapic-generator-java/",\n)' WORKSPACE
+if [[ $download_repos -eq 1 ]]; then
+  bash download-repos.sh
```

Comment:
I suppose we can move the download and delete repo logic into the `generate-all.sh` so it's more efficient. 
Then in future, when we have the metadata as discussed today, we can checkout specific commitish here for each client library potentially.

Reply:
Oh, I'm actually downloading the repos _once_ in `generate-all.sh:8`. This piece of code was intended in case we want to call `generate-one.sh` directly and it's not reached when this script is called from `generate-all.sh`.